tox.ini

```diff
@@ -4,8 +4,6 @@ envlist = py37,flake8
 [testenv]
 deps =
     -rrequirements.txt
-    pytest
-commands = {envpython} -m pytest
```

Comment:
Dropping it like this will disable (aka break) unit test run. We should just reconfigure it so that tox still runs unit tests, by adding parameters like `--ignore=tests` or `-c` (if we need custom pytest.ini for unittests):

```
commands = {envpython} -m pytest  --ignore=tests -c unittests/pytest.ini unittests
```

Reply:
I agree, but as we don't have any unittests related to our changed ocsci, I've just disabled this till we will have some and then we can re-enable, but yeah. I can do what you have suggested.

Reply:
done